content/en/blog/2023/jaeger-exporter-collector-migration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Migrating away from the Jaeger exporter in the Collector
+linkTitle: Migrating away from the Jaeger exporter in the Collector
+date: 2023-09-14
```

Comment:
Let's aim for some time next week, we still have another blog post in the loop for Monday next week, maybe Wednesday/Thursday?

Reply:
That's fine, the release was done Monday and there was an issue raised yesterday about it, which is why I figured the sooner the better :)